data.md

```diff
@@ -0,0 +1,107 @@
+# How We Use Data
+
+_The outline and format of this page is adopted from [Projects by IF: How IF uses data](https://www.projectsbyif.com/how-if-uses-data/)._
```

Comment:
```suggestion
_This page's outline and format is adopted from [Projects by IF: How IF uses data](https://www.projectsbyif.com/how-if-uses-data/)._
```
Would move this below the header matter, typically we've handled crediting not at the top